app/views/pages/planner.html.erb

```diff
@@ -154,11 +164,27 @@
 </div>
 
 <script>
+
   if (localStorage['scroll']) {
     window.scroll({ top: localStorage['scroll'] })
   }
   window.addEventListener('scroll', () => {
     localStorage['scroll'] = window.scrollY;
   });
+
+  var scrollable = document.querySelector('.planner-day');
+
+  scrollable.addEventListener('wheel', function(event) {
+    var deltaY = event.deltaY;
+    var contentHeight = scrollable.scrollHeight;
+    var visibleHeight = scrollable.offsetHeight;
+    var scrollTop = scrollable.scrollTop;
+
+    if (scrollTop === 0 && deltaY < 0)
+      event.preventDefault();
+    else if (visibleHeight + scrollTop === contentHeight && deltaY > 0)
+      event.preventDefault();
+  });
+
```

Comment:
Is this the bit of code that isn't quite working yet in terms of setting the start point of the scroll at a certain point down the line?

Reply:
soz, missed this before!

so...when you're scrolling from within the planner, when you've reached the top or bottom limit of the grid, the main webpage behind starts scrolling up and down instead until it reaches the end of its boundaries, which can be distracting/annoying/non-UX-friendly. this code prevents the main webpage from scrolling once you hit the top/bottom limits of the planner.

i havent found an easy, working solution to setting the start point of the scroll at a certain point down the planner. so now i have to stop being lazy and actually think! might try storing the div class as a JS variable, put an event listener on it, and then set the y position/coordinate of it as not 0? or find a way to make that 'inner' planner scroll stick (like we have on eg Shows page). tried it by tweaking Lucien's JS and subbing in 'scrollable' for 'window' but it didnt quite work